vhost-device-scmi/src/main.rs

```diff
@@ -126,6 +126,11 @@ fn start_backend(config: VuScmiConfig) -> Result<()> {
         )
         .unwrap();
 
+        // Register devices such as "/dev/iio:deviceX" which can initiatively notify frontend to epoll handler.
```

Comment:
"..., which can actively notify the frontend to epoll the handler." ?

Reply:
well